lib/mayaHydra/hydraExtensions/sceneIndex/mayaHydraSceneIndex.cpp

```diff
@@ -773,26 +769,20 @@ VtValue MayaHydraSceneIndex::GetMaterialResource(const SdfPath& id)
     return ret.IsEmpty() ? MayaHydraMaterialAdapter::GetPreviewMaterialResource(id) : ret;
 }
 
-//Create the default material from the "standardSurface1" maya material or create a fallback material if it cannot be found
+//Create the default maya material or create a fallback material if it cannot be found
 void MayaHydraSceneIndex::CreateMayaDefaultMaterialData() 
 { 
-    // Try to get the standardSurface1 material
-    MObject defaultShaderObj;
-    GetDependNodeFromNodeName("standardSurface1", defaultShaderObj); // From mayautils.cpp
-    bool defaultMaterialSuccessfullyCreated = false;
-    if (MObjectHandle(defaultShaderObj).isValid()) {
-        //Get its shading group as it is what we use to create a material adapter
-        MObject defaultMaterialShadingGroupObj
-            = GetShadingGroupFromShader(defaultShaderObj); // From mayautils.cpp
-        if (MObjectHandle(defaultMaterialShadingGroupObj).isValid()) {
-            defaultMaterialSuccessfullyCreated = _CreateMaterial(MayaHydraSceneIndex::_mayaDefaultMaterialPath, defaultMaterialShadingGroupObj);
-        }
-    }
-
-    if (! defaultMaterialSuccessfullyCreated){
-        TF_CODING_WARNING("standardSurface1 material and its shading group could not be retrieved, using a fallback material");
-        // In case we could not create the default material from the standardSurface1 material, we
-        // create a fallback material
+    bool defaultMaterialCreatedSuccessfully = false;
+
+    MObject defaultMaterialShadingGroupObj = getDefaultMaterialShadingGroupNode();
```

Comment:
I recall there's already an Maya API (MMaterial::defaultMaterial()) to get the default Material, no need to query again. Just use MObject defaultMaterialShadingGroupObj = MMaterial::defaultMaterial().shadingEngine();

Reply:
here's the API doc:
https://help.autodesk.com/view/MAYAUL/2024/ENU/?guid=MAYA_API_REF_cpp_ref_class_m_material_html

static MMaterial defaultMaterial()

Get the default material.
There will always be a default material in the scene and therefore the result of this function should always succeed. The default material will correspond to the initialShadingGroup node that is in the scene.